apps/site/assets/css/_events.scss

```diff
@@ -324,10 +320,21 @@ $mobile-control-height: 73px;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
+
+      &:hover, &:focus {
+        color: $brand-primary;
+        text-decoration: none;
+      }
     }
   }
 }
 
+button[data-js='enabled'] {
```

Comment:
Would it work to target the `data-a11y-dialog-show` attribute (e.g. `.m-event-calendar__event[data-a11y-dialog-show]` to select it regardless of value) rather than adding a new attribute for this?

Reply:
Yes, that looks a lot better 🙏